src/logcollector/logcollector.c

```diff
@@ -738,8 +717,6 @@ void LogCollectorStart()
             f_check = 0;
         }
 
-        rand_keepalive_str(keepalive, KEEPALIVE_SIZE);
```

Comment:
As a result of the deletion of every call to `KEEPALIVE_SIZE` this define is left unused. It would be nice to delete it too:
https://github.com/wazuh/wazuh/blob/3cf21b428994459a622eed54d70f4f84902edd29/src/headers/defs.h#L59